sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -521,11 +522,11 @@ private case class GpuParquetFileFilterHandler(@transient sqlConf: SQLConf) exte
         if (dt == DataTypes.LongType && pt.getOriginalType == OriginalType.UINT_32) {
           return
         }
-        // TODO: enable below matches after we support downcast INT32 during parquet reading
-        //  https://github.com/NVIDIA/spark-rapids/issues/5445
-        // if (dt == DataTypes.ByteType || dt == DataTypes.ShortType || dt == DataTypes.DateType) {
-        //   return
-        // }
+        // TODO: Add below converters for INT32. Converters work when evolving schema over cuDF
+        //  table read from Parquet file. https://github.com/NVIDIA/spark-rapids/issues/5445
+         if (dt == DataTypes.ByteType || dt == DataTypes.ShortType || dt == DataTypes.DateType) {
```

Comment:
I uncommented this combinations because the UT cases of parquet writing fails if it is disabled. I will fix #5445 ASAP.